examples/interactive2-layer-and-step-filtering.fixture.tsx

```diff
@@ -4,6 +4,7 @@ import exampleGraphics from "site/assets/exampleGraphics.json"
 export default () => {
   return (
     <InteractiveGraphics
+      objectLimit={2}
       graphics={{
```

Comment:
Dont break an existing fixture just to test something new, create a new fixture here